distribution/linux/openrct2.appdata.xml

```diff
@@ -63,4 +63,8 @@
     <content_attribute id="money-purchasing">none</content_attribute>
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
+  <releases>
+      <release version="0.2.6" date="2020-04-17" />
```

Comment:
So if I read the documentation correctly, to add the release notes, we could add a tag inside that reads as follows?
```<url>https://github.com/OpenRCT2/OpenRCT2/releases/tag/v0.2.6</url>```

Reply:
That's how I read it too. Would you like me to add that?

Reply:
Yes, please. And also for 0.2.5. 

You can leave out the other versions if you prefer.

Reply:
OK, I've gone back as far as 0.2.0, which seemed like a more coherent starting point.